src/js/providers/tracks-mixin.js

```diff
@@ -60,19 +63,20 @@ function setTextTracks(tracks) {
     } else {
         // Remove the 608 captions track that was mutated by the browser
         this._unknownCount = 0;
-        this._textTracks = this._textTracks.filter(function(track) {
+        this._textTracks = this._textTracks.filter((track) => {
             const trackId = track._id;
             if (this.renderNatively && trackId && trackId.indexOf('nativecaptions') === 0) {
                 delete this._tracksById[trackId];
                 return false;
             } else if (track.name && track.name.indexOf('Unknown') === 0) {
                 this._unknownCount++;
             }
+            if (trackId.indexOf('nativemetadata') === 0 && track.inBandMetadataTrackDispatchType === 'com.apple.streaming') {
```

Comment:
can i get a quick explanation of this?

Reply:
Metadata TextTracks created by Safari have an `id` that starts with 'nativemetadata' (if they create a second track it's called 'nativemetadata1') and an `inBandMetadataTrackDispatchType` of 'com.apple.streaming'.

It was incorrect to assume there would only be one `nativemetadata` track, and that it would be the one added by Safari to handle ID3. We insert our own `nativemetadata` track before Safari, so the track it creates has an id of 'nativemetadata1'.